### DISCUSSION TITLE
force:source:push for Mono repo with MDP

### DISCUSSION THREAD

**Discussion by honeyverma** [forcedotcom/cli#2353] [Ideas]
**Is your feature request related to a problem? Please describe.**
We are trying to create a mono repo for the unlocked package and it's almost done. However, source:push doesn't take a specific directory. So our mono repo would look like this:

main-dir
- .sfdx/
- pkg-common/
- pkg-abc/
- pkg-pqr/
- pkg-xyz/
- sfdx-project.json

here pkg-common|abc|pqr|xyz are 4 unlocked packages, and all of them are independent, we want to keep them separate, we would like to enforce engineers to work on one package at a time, to maintain the inline track. And its all set, however, the only problem we are facing with `force:source:push` is we cannot specify the pkg directory it would be better if we can have something like this

`force:source:push --pkgDir pkg-abc` 

where --pkgDir could be optional for a developer

[a comment by an automated account is not shown]

**Comment by mshanemc** [comment]
`project deploy start --source-dir` solves this for you.
